test/e2e/specs/editor/blocks/cover.spec.js

```diff
@@ -200,7 +194,7 @@ test.describe( 'Cover', () => {
 		expect( newCoverBlockBox.height ).toBe( coverBlockBox.height + 100 );
 	} );
 
-	test( 'dims the background image down by 50% when transformed from the Image block', async ( {
+	test( 'dims the background image down by 50% black when transformed from the Image block', async ( {
```

Comment:
I specified black here because it isn't possible to run FAC in a transformation due to it being asynchronous and it may have been confusing that it isn't getting the average color like the other test.